zubhub_frontend/zubhub/src/components/input/input.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import clsx from 'clsx'
+import {  
+    FormControl,  
+    OutlinedInput,
+    FormHelperText,
+} from '@material-ui/core';
+import { handleTextFieldChange, handleTextFieldBlur} from '../../views/create_activity/createActivityScripts';
+
+function Input(props) {
+    const {
+      label,
+      defaultValue,
+      classes, 
+    }=props
+  return (
+    <div>
+      <FormControl
+        className={clsx(classes.margin, classes.textField)}
+        variant="outlined"
+        size="small"
+        fullWidth
+        margin="none"
+        error={props.errors[label]? true: false}
+        >  
+        <OutlinedInput
+        className={classes.customInputStyle}
+        id={label}
+        name={label}
```

Comment:
please do not use label as name, Please add another property or user property key

Reply:
ok I understand, I will change this property name to key of fieldName